### PR TITLE
Adds description elements to charts.

### LINKED ISSUE #1011: add support for descriptions on chart components
currently, we have the ability to label chart components, but not to add long-text descriptions to them.

the solution would be to add `<desc>` elements alongside the already existing `<text>` elements to the SVG, and then fill them with passed in data.
the descriptive data would most likely be the texts that are currently rendered on the tool tips. 

this should help in making our charts more accessible for visually impaired users.

see https://developer.mozilla.org/en-US/docs/Web/SVG/Element/desc

### PR DESCRIPTION
**Pull request by stopfstedt**
fixes #1011

### CODE REVIEW

**Comment by stopfstedt** [comment]
@jrjohnson side-note: i left the color box chart out since i don't really know if a description here is even applicable. all the other ones are pretty clear-cut. pls let me know if you have any thoughts on that one.

File: packages/test-app/tests/integration/components/simple-chart-pack-test.js [hunk 1]
```diff
@@ -21,10 +21,14 @@ module('Integration | Component | simple chart pack', function (hooks) {
       @containerHeight="100%"
       @containerWidth="100%"
     />`);
+    //await pauseTest();
```
Comment (jrjohnson) [inline packages/test-app/tests/integration/components/simple-chart-pack-test.js:24]: //

**Review by jrjohnson — CHANGES_REQUESTED** [review]
I love this!

Can you also add this new chart data to the docs app. I'm not sure we need special documentation for this, but I think the chart data from the test app will make this feature more discoverable and easier to manually test since the docs app gets a preview rendering.

File: packages/test-app/tests/integration/components/simple-chart-pack-test.js (hunk 1 above)
Comment (stopfstedt) [reply packages/test-app/tests/integration/components/simple-chart-pack-test.js:24]: handled.

**Comment by stopfstedt** [comment]
> I love this!
> 
> Can you also add this new chart data to the docs app. I'm not sure we need special documentation for this, but I think the chart data from the test app will make this feature more discoverable and easier to manually test since the docs app gets a preview rendering.

:100:  will update the docs app now.

**Review by jrjohnson — APPROVED** [review]

**Pull request merged**